microcosm_flask/swagger/definitions.py

```diff
@@ -35,6 +35,7 @@
 
 
 logger = getLogger("microcosm_flask.swagger")
+PLACEHOLDER_INTEGER_ID = 1111
```

Comment:
Any reasons for this particular value or does it just need to be suitably large?

Reply:
I wanted something that wouldn't already be in the URI by chance.